project_merge/wizard/project_task_merge_views.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+        <record id="project_task_merge_view_form" model="ir.ui.view">
+            <field name='name'>project.task.merge.form</field>
+            <field name='model'>project.task.merge</field>
+            <field name='arch' type='xml'>
+                <form string='Merge Tasks'>
+                    <sheet>
+                        <group>
+                            <group>
+                                <field name="user_ids" widget="many2many_tags" />
```

Comment:
Functional suggestion: Would be nice if the many2many_tags_avatar widget is used here.

Reply:
Updated the widget